docs/user-signature.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Signing and Verifying Arbitrary Data
+---
+
+## Signing Arbitrary Data
+
+Cryptographic signatures are a key part of the blockchain. They are used to prove ownership of an address without exposing its private key.
+This is primarily used for signing transactions but can also be used to sign arbitrary messages.
+
+FCL has a feature that lets you send arbitrary data to a configured wallet/service where the
+user may approve signing it with their private key/s.
+
+## Verifying User Signatures
+
+What makes message signatures more interesting is that we can use Flow blockchain to verify the ECDSA signatures.
```

Comment:
Dont think we need to mention ECDSA here. Nothing in Flow is technically limited to ECDSA